common/artifacts/909_wc_artifacts_library_librams.txt

```diff
@@ -6,6 +6,9 @@ libram_of_protection = {
 	active = {
 		is_paladin_class_trigger = yes
 	}
+	allowed_gift = {
+		religion_group = light_group
```

Comment:
```suggestion
		is_paladin_class_trigger = yes
```
...since we may allow more religions to have paladins soon: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/1221.

Reply:
ok, didn't keep up with all suggestions

Reply:
done